README.md

```diff
@@ -51,16 +51,42 @@ The following environment variables are expected
 - `CASUAL_CALLER_CONNECTION_FACTORY_JNDI_SEARCH_ROOT`, String, default value "eis"
 - `CASUAL_CALLER_VALIDATION_INTERVAL`, int, default value 5000
 - `CASUAL_CALLER_TRANSACTION_STICKY`, boolean, default value false
+- `CASUAL_CALLER_TOPOLOGY_CHANGED_DELAY`, long, defaults to 50 (ms)
+- `CASUAL_CALLER_SERVICE_ROUTES_FILE`, String, default - not set
 
 A typical config file can look like the following:
 ```json
 {
   "jndiSearchRoot": "somecustomjndiroot",
   "validationIntervalMillis": 500,
-  "transactionStickyEnabled": true
+  "transactionStickyEnabled": true,
+  "topologyChangeDelayMillis": 5000,
+  "routeFileName": "foo.json"
 }
 ```
 
+### HTTP Route Configuration
+
+If you have services that you want to be called non transactional via http then you can define them in a ```route.json``` file such as:
+```json
+{
+  "routes":
+  [
+    { "name": "test-service", "uri": "http://test-host.test:9990/test-service"},
+    { "name": "another-test-service", "uri": "http://test-host.test:9990/another-test-service"}
+  ]
+}
+```
+
+Using the example above, when a tpcall is issued to service `test-service`, instead of doing the tpcall a http request is dispatched to `http://test-host.test:9990/test-service`.
```

Comment:
I assume this relies on the http app you made, but you have not referenced that here?
Additionally, i would argue that a client for that should have been made within that repository and published, which can be utilised within this repository to reduce the work required here allowing you to just mock that imported client in this code.
I also feel we are making too many repositories with very tightly intertwinded dependencies to each other.

Reply:
No it does not, it has nothing to do with this at all.

Reply:
This code makes so that you can decide that tpcalls to wherever, casual or java appserver, should be made using http.

The http application *can* be used to expose such casual services from a java appserver if you so wish.

Reply:
I am lost. 
But, if you want to call via http a java app server service, it would need the new http application then, right?
Or is this only intended for calls from java to native?
I thought this was to provide the final symmetry for casual-java to http based tpcall requests? So you can call inbound or outbound through java.